docs/en/latest/plugins/ip-restriction.md

```diff
@@ -1,5 +1,11 @@
 ---
 title: ip-restriction
+keywords:
+  - APISIX
+  - Plugin
+  - IP restriction
+  - ip-restriction
+description: This document contains information about the Apache APISIX ip-restriction Plugin.
```

Comment:
We could even improve this section with some semantic keywords here

Reply:
Yes. I'm currently keeping it similar. But after we have done some research on what works best, we can definitely change them.